Src/library/net/abstract/socket/socket.e

```diff
@@ -1041,42 +1126,6 @@ feature {NONE} -- Implementation
 		deferred
 		end
 
-feature -- Socket Recv and Send timeout.
```

Comment:
To be coherent with other similar routines that are in the descendant ...
such as the `timeout` 
But I don't mind keeping them in SOCKET